### PR TITLE
Gilmatan

### PR DESCRIPTION
**Pull request by gilyaakov**

### CODE REVIEW

File: doc/HOW_TO _RUN_BUILDL.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Welcome to Explanation of the bash script buildl.sh 
+## (1) First lets take a look in the usage of the script:
+		usage: ./buildl.sh [test_0] [test_1] ... [test_n]
```
Comment (amichai-bd) [inline doc/HOW_TO _RUN_BUILDL.md:3]: Your editor is using TABs?
Please make sure its changed to Spaces.
See what it looks like here:(bad alignment).
image

File: modelsim_run/work/_info [hunk 2]
```diff
@@ -10,45 +10,45 @@ cModel Technology
 Z0 dC:/Users/gilya/OneDrive/Desktop/studies/4th/project/FinalProjectWS/rvc_asap/modelsim_run
 vrvc_asap
 Z1 DXx6 sv_std 3 std 0 22 VYECXdT12H8WgbUP_5Y6:3
-Z2 DXx4 work 12 rvc_asap_pkg 0 22 OHOhSK@HQ:AT48egM@m^B3
-Z3 !s110 1639590809
+Z2 DXx4 work 12 rvc_asap_pkg 0 22 ?Z_PPeMaSfI2Q7UjAX[:z2
```
Comment (amichai-bd) [inline modelsim_run/work/_info:13]: Please dont Commit this file.
Add to git ignore

**Review by amichai-bd — APPROVED** [review]
Please read review comment

File: modelsim_run/work/_info (hunk 2 above)
Comment (gilyaakov) [reply modelsim_run/work/_info:13]: This whole folder in the .gitignore, I tried but can not get rid of this file.

File: doc/HOW_TO _RUN_BUILDL.md (hunk 1 above)
Comment (gilyaakov) [reply doc/HOW_TO _RUN_BUILDL.md:3]: Apparently this is a problem showing up here on github, it looks just fine in my environment.

Comment (amichai-bd) [reply doc/HOW_TO _RUN_BUILDL.md:3]: That is exactly why you should use  spaces and not tabs!! So it looks the same in all environment!!

Please remove the tabs and use spaces

Comment (amichai-bd) [reply doc/HOW_TO _RUN_BUILDL.md:3]: That is exactly why you should use  spaces and not tabs!! So it looks the same in all environment!!

Please remove the tabs and use spaces

Comment (gilyaakov) [reply doc/HOW_TO _RUN_BUILDL.md:3]: Okay, I got it. The issue will be addressed.

**Pull request merged**